recipe/meta.yaml

```
@@ -1,69 +1,77 @@
 {% set name = "awscli" %}
-{% set version = "2.24.12" %}
+{% set version = "2.27.49" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
-# v2 is not on PyPI https://github.com/aws/aws-cli/issues/6785
 source:
   url: https://github.com/aws/aws-cli/archive/{{ version }}.tar.gz
-  sha256: 32f4006e4c874972f1d3ac41144788b93ea9febe5d68d029507ef5eab9260c65
+  sha256: a8043443d3d01b7857b62f67799de0d5ed4ae0bb413d51229f33a4c32126e686
+  patches:
+    - patches/0001-fix-dependencies-version.patch
 
 build:
   number: 0
-  # Build uses pep517 with a custom backend
-  # console_scripts entrypoints aren't used, so this can't be noarch
   script:
     - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv
     - mkdir -p $PREFIX/share/bash-completions/completions && echo complete -C aws_completer aws >$PREFIX/share/bash-completions/completions/aws  # [unix]
+  skip: True  # [py<39]
 
 requirements:
   build:
-    - python
+    - patch  # [not win]
+    - m2-patch  # [win]
```

```suggestion
    - msys2-patch  # [win]
```
`msys2-patch` is a modern MSYS2 package. `m2-` are outdated and should be replaced by `msys2-` in the `build` section.